### ISSUE TITLE
Add the Text output for GeminiMultimodalLiveLLMService

### ISSUE THREAD

**Issue by fatwang2** [pipecat-ai/pipecat#892]
# Description
Is this reporting a bug or feature request?
feature

Will you think about change the response_modalities of the output of Gemini, so we can use the output to other tts service.

It will output by text only if you change it to TEXT.

**Comment by markbackman** [comment]
@fatwang2 can you clarify? Do you want audio input and text output?

**Comment by fatwang2** [comment]
yes, so that I can transfer to other TTS to support multi languages, I have
finished it on my own git

Mark Backman ***@***.***>于2024年12月22日 周日22:49写道：

**Comment by golbin** [comment]
I also wish for this feature. I believe it’s very powerful and represents the future.

**Comment by markbackman** [comment]
https://github.com/pipecat-ai/pipecat/pull/926

Thanks to @imsakg for the PR! 🙌
